### PR TITLE
Allow filtering out features from code external to the current package.

### PR DESCRIPTION
**Pull request by rictic**
Builds on top of #449 

 - [x] CHANGELOG.md has been updated

### CODE REVIEW

File: src/model/package.ts [hunk 1]
```diff
@@ -17,6 +17,18 @@ import {Document, FeatureKinds} from './document';
 import {Feature} from './feature';
 import {Queryable} from './queryable';
 
+export type QueryOptions = object & {
+  /**
+   * If true then results will include features from outside the package, e.g.
+   * from files in bower_components or node_modules directories.
+   */
+  includeExternalCode?: boolean;
```
Comment (justinfagnani) [inline src/model/package.ts]: I'd suggest `includeExternalLibs`, `externalLibs`, `includePackages`, or just `packages`. It'd be nice to align this with the `imports` option name, even if they're not now used together, or never will.

Comment (rictic) [inline src/model/package.ts]: What about `external`?

Comment (rictic) [inline src/model/package.ts]: Agreed on `externalPackages` on slack.

**Comment by rictic** [comment]
Made `externalPackages` an option on Document queries too.

**Comment by justinfagnani** [comment]
Cool.

I wonder if ultimately we want to simplify the get & query API to just:

```typescript
interface query {
  id?: string,
  kind?: string,
  imports?: boolean,
  externalPackages?: boolean,
  inlineDocuments?: boolean,
}

getFeature(query: Query): Set<Feature>;
```

**Pull request merged**